engine/gamesys/src/gamesys/test/resource/set_texture.script

```diff
@@ -0,0 +1,97 @@
+-- Copyright 2020-2022 The Defold Foundation
+-- Copyright 2014-2020 King
+-- Copyright 2009-2014 Ragnar Svensson, Christian Murray
+-- Licensed under the Defold License version 1.0 (the "License"); you may not use
+-- this file except in compliance with the License.
+--
+-- You may obtain a copy of the License, together with FAQs at
+-- https://www.defold.com/license
+--
+-- Unless required by applicable law or agreed to in writing, software distributed
+-- under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+-- CONDITIONS OF ANY KIND, either express or implied. See the License for the
+-- specific language governing permissions and limitations under the License.
+
+function init(self)
+	self.update_counter = 0
+end
+
+-- tile_set of sprite is 84 x 67
```

Comment:
Fyi: the _original_ dimensions are 84, 67 - the actual texture gets resized to 64,64 - I should change this comment